README.md

```diff
@@ -1,2 +1,37 @@
 # crispy
 Docs/Links/Infrastructure. Successor to database management tools
+
+## Overview 
+
+This is a tool to make developers life easier and database setup more reliable.
+
+## Architecture
+
+### Components
+
+#### crispy 
+
+  - https://github.com/WiseEngineering/crispy-worker
+  - https://github.com/WiseEngineering/crispy-web
+  - https://github.com/WiseEngineering/crispy-api
+
+They must manage all database operations and be able to execute them in a particular worker. 
+The main goal for those Components to be responsible for running database operations in `worker` and give the user the ability to manage it from [WEB](https://github.com/WiseEngineering/crispy-web)
+
+#### crispy-cli
+
+https://github.com/WiseEngineering/crispy-cli
+
+Simple CLI interface where you are able to sync your migrations with `crispy` Component.
+
+## How components will interact with each other
+
+- [crispy-web](https://github.com/WiseEngineering/crispy-web) communicates with [crispy-api](https://github.com/WiseEngineering/crispy-api)
+
+ - [crispy-web](https://github.com/WiseEngineering/crispy-web) communicates with [crispy-worker](https://github.com/WiseEngineering/crispy-worker)
+ 
+- [crispy-cli](https://github.com/WiseEngineering/crispy-cli) communicates with migrations folder/library based on adapter
+
+- [crispy-cli](https://github.com/WiseEngineering/crispy-cli) communicates with [crispy-api](https://github.com/WiseEngineering/crispy-api)
```

Comment:
I still need more details about `crispy-cli` to better understand it. Should it be an alternative to web UI - then it should communicate to the `api` only? Should it be a layer between worker and external migration repo - then no calls to api?

Reply:
The point about `crispy-cli` is open to discussion.

Thinking about `crispy-cli` as a tool to handle:
- API communication as an alternative for web UI.
- send info about migrations from migrations repo

The most critical question here is how we connect `crispy-cli` with already implemented libraries.
The main problems there :
-  we cannot use a specific folder structure as users can configure migrations libraries in their own, and we cannot simply parse them
- we cannot parse sql commands as they also can be written differently. 
- didn't see the solution how we can parse migrations  based on models

Reply:
> API communication as an alternative for web UI.
Agree, this is a common practice, though it might have not the highest priority.

> The most critical question here is how we connect crispy-cli with already implemented libraries.
Yes, it's a tuff question. Maybe leave CLI in a 'grey' area for this PR and we will iterate how to deal with the listed problems in a follow up by discussing specific examples.

Reply:
Added issue to think
https://github.com/WiseEngineering/crispy-cli/issues/2